README.es.md

```diff
@@ -48,11 +48,11 @@ Histórico de modificaciones:   2009-05-06 RSB  Transcrito a partir de imágenes
 
 Este código fuente se ha transcrito o adaptado de imágenes digitalizadas
 de una copia en papel presente en el museo del MIT. Los empleados del museo
-Paul Fjeld y Deborah Douglas relizaron la digitalización y la gestión de las
+Paul Fjeld y Deborah Douglas realizaron la digitalización y la gestión de las
 imágenes respectivamente. Muchas gracias a ambos. Las imágenes (con la
 consecuente reducción de su tamaño para almacenamiento y de su calidad) están
 disponibles en línea en la dirección www.ibiblio.org/apollo. Si por alguna razón
-considera que algunas imágenes son ilegibles, contacte comigo en info@sandroid.org
+considera que algunas imágenes son ilegibles, contacte conmigo en info@sandroid.org
```

Comment:
Is that really how it's said? I find it natural to say `contáctame por <email>` or `contactar con <nombre/email>`

Reply:
The first option is pretty similar with the correction, and indeed we usually use "contáctame" in our daily speaking, but since I can perceive that certain formality was used (due to "Contacte conmigo"), I suggest to modify the second proposed option to "Contactar a <nombre/email>", What do you think?